tratto/src/main/java/star/tratto/util/javaparser/JavaParserUtils.java

```diff
@@ -462,6 +614,39 @@ public static String getMethodSignature(MethodDeclaration methodDeclaration) {
         return method.trim().replaceAll(";$", "");
     }
 
+    /**
+     * Get all generic types of a given method.
+     */
+    private static List<String> getMethodUsageTypeParameters(MethodUsage methodUsage) {
+        return methodUsage.getDeclaration().getTypeParameters()
+                .stream()
+                .map(ResolvedTypeParameterDeclaration::getName)
+                .toList();
+    }
+
+    /**
+     * Get all parameters in the method arguments.
```

Comment:
For clarity, I like to say "formal parameter" or "type parameter".  Here, I think it's formal parameters.